src/main/kotlin/com/sourcegraph/cody/statusbar/CodyAutocompleteStatusService.kt

```diff
@@ -38,9 +38,11 @@ class CodyAutocompleteStatusService : CodyAutocompleteStatusListener, Disposable
 
   override fun onCodyAutocompleteStatusReset(project: Project) {
     ApplicationManager.getApplication().executeOnPooledThread {
-      val notify = didStatusChanged(project)
-      if (notify) {
-        updateCodyStatusBarIcons()
+      if (!project.isDisposed) {
```

Comment:
Another issue - the same scenario (a quick open & close project) - this one is (almost) deterministic. Should not be a case anymore.